recipes/yaml-cpp/all/conanfile.py

```diff
@@ -32,7 +32,9 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.compiler.cppstd:
+        if self.settings.compiler == "Visual Studio" and self.settings.compiler.version == "12":
+            raise Exception("Visual Studio 12 not supported: Library needs C++11 standard")
+        elif self.settings.compiler.cppstd:
```

Comment:
Isn't this already covered by `tools.check_min_cppstd(self, "11")` below ?

Reply:
I read [your comment](https://github.com/conan-io/conan-center-index/issues/2525#issuecomment-673991062) after creating this PR. Sorry for disturbing!